examples/hellogrpc/e2e-test.sh

```diff
@@ -55,9 +55,14 @@ check_msg() {
 
   while [[ -z $(get_lb_ip) ]]; do
     echo "service has not yet received an IP address, sleeping for 5s..."
-    sleep 5
+    sleep 10
   done
 
+  # Wait some more for after IP address allocation for the service to come
+  # alive
+  echo "Got IP Adress! Sleeping 30s more for service to come alive..."
+  sleep 30
```

Comment:
We have had to loop multiple times waiting on LBs.  I would recommend putting in a couple of 30 second loops.